kuma/payments/forms.py

```diff
@@ -213,6 +213,16 @@ def make_charge(self):
 
 
 class RecurringPaymentForm(ContributionForm):
+    accept_checkbox = forms.BooleanField(
+        label=u'',
+        required=True,
+        widget=forms.CheckboxInput(
+            attrs={
+                'class': 'required checkbox form-control',
+                'data-error-message': _('Must be more than $1')
```

Comment:
This message should be changed to something like ``_('You must agree to the terms to continue')``. I think it is impossible without developer shenanigans to get the message, however.

Reply:
👍 fixed, and now displaying errors on the form.